molecule/default/converge.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Converge
+  hosts: all
+  become: true
+
+  pre_tasks:
+  - name: Update apt cache.
+    apt: update_cache=yes cache_valid_time=600
+    when: ansible_os_family == 'Debian'
+
+  roles:
+  - role: cvmfs_contrib.cvmfs_client
```

Comment:
should it be `ansible-cvmfs-client` ?

Reply:
If you ever want to publish this role on ansible galaxy, you will need to switch to underscores.

Having hyphens in the repository name and underscores in the role name is common.
It is also common to include the `ansible-` prefix in the repository name and to remove it for the role name (all roles on ansible galaxy are ansible roles...).

In summary, this is the name I would suggest.